### PR TITLE
feat(Use Skill): add event triggered by ember skill use

### PR DESCRIPTION
**Pull request by mannyovena**

### CODE REVIEW

**Comment by gruddy** [comment]
What sort of access control options do we have for this? Ideally the consumption pattern is scoped to lsitening to your own channel's activity. Is that feasible?

**Comment by connor4312** [comment]
Let's chat irl. Richard will set up a meeting.

File: pages/02.reference/02.constellation/02.events/02.live/29.Use Skill/doc.md [hunk 1]
```diff
@@ -1,20 +1,21 @@
...
 ## Examples
 
 ```json
 {
-	"triggeringuserid": 12345,
-	"currency": "embers",
-	"amount": 50,
-	"skillid": "3c8f4d71-1736-4e03-8e7b-13bca00cbf82"
+        "triggeringUserId": 12345,
+        "currency": "embers",
```
Comment (mannyovena) [inline pages/02.reference/02.constellation/02.events/02.live/29.Use Skill/doc.md]: @ProbablePrime This should be `currencyType`

**Comment by ProbablePrime** [comment]
Closing this for now to avoid confusion.

**Pull request closed without merging**